.github/workflows/basic.yaml

```diff
@@ -42,7 +44,7 @@ jobs:
 
   cosmwasm-compilation:
     name: Cosmwasm Compilation
-    runs-on: ubuntu-8-core-32-gb
+    runs-on: blacksmith-16vcpu-ubuntu-2204
```

Comment:
How is this thing faster yet requires more CPUs than previously set ubuntu?

Reply:
While blacksmith 8 core is already [faster](https://github.com/BlacksmithOSSTests/axelar-amplifier/actions/runs/7892299883/job/21538883237?pr=1), we can run on even more cores for less price than github 8 core cpu. Updated readme, and see the pricing here: https://docs.blacksmith.sh/runners/pricing